spec/acceptance/basic_spec.rb

```diff
@@ -76,6 +76,7 @@ class { 'pulpcore':
 
   describe command("PULP_SETTINGS=/etc/pulp/settings.py pulpcore-manager diffsettings") do
     its(:stdout) { is_expected.to match(/^USE_NEW_WORKER_TYPE = True/) }
+    its(:stdout) { is_expected.to match(/^WORKER_TTL = 30/) }
```

Comment:
The point of the test was to catch when Pulp's default changes so that Pulp's default in the param doc could be updated. If we don't document Pulp's default then there is no point to the test, so I'll remove it.